assets/js/googlesitekit-modules-thank-with-google.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Subscribe with Google module entrypoint.
+ * Thank with Google module entrypoint.
  *
  * Site Kit by Google, Copyright 2021 Google LLC
```

Comment:
@kuasha420 Since we are updating the `@since` annotations to treat the code as new, I think we should update the year across all the TwG files from `2021` to `2022`

Reply:
We had a similar situation previously on a ETS ticket where we were bringing back old code that was already part of the repo the consensus was to use the initial year when the code materialized for the copyright year. Hence I kept it unchanged for this and #5391.

Cheers.

Reply:
Thanks @kuasha420 !